server/controllers/auth/login.js

```diff
@@ -0,0 +1,85 @@
+// modules
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const database = require("../../database");
+
+exports.login = (req, res, next) => {
+  // todo: 'remember me' feature
+  const email = req.body.email;
+  const password = req.body.password;
+
+  // find member using email address
+  database.query(`
+    SELECT
+      member_id, first_name, last_name, email_address, password, profile_picture, is_owner, is_moderator, is_blocked, created_at
+    FROM
+      member
+    WHERE
+      email_address = '${email}'
+  ;`).then(databaseResult => {
+    // get first member from array of objects
+    const memberData = databaseResult.rows[0];
+
+    // validate member password with encrypted password in database
+    bcrypt.compare(password, memberData.password).then(validatePassword => {
+      if (validatePassword) {
+
+        const loginToken = jwt.sign({
+          member_id: memberData.member_id,
+          email_address: memberData.email_address,
+          is_owner: memberData.is_owner,
+          is_moderator: memberData.is_moderator,
+          is_blocked: memberData.is_blocked,
+          created_at: memberData.created_at
+        }, 'secretKey', { expiresIn: '2d' });
```

Comment:
It's better to reference secrete key from `.env` file rather than hard coding it.

Reply:
On that topic, I did posted on site for getting thoughts from other developers about it.

https://hashnode.com/post/can-we-use-user-password-as-jwt-secret-key-ck5vzwvmv0894qks1978ib9gh

Reply:
Dude, that's an awesome idea! I never considered that! 😆

Reply:
_yeah, I still need to reply to their comments, so let's keep it for now._

Let's hear what other devs have to say and then we can make changes accordingly.